hibp/client.go

```diff
@@ -62,3 +62,31 @@ func createClient(ctx context.Context, apiKey string) *hibp.Client {
 	cl := hibp.New(clientOptions...)
 	return &cl
 }
+
+// Create a client without an API key
+// Certain tables like hibp_password do not require an API key
```

Comment:
What happens if I create a client without an API key and query a table that requires one? What error message is returned?